decision/12.md

```diff
@@ -0,0 +1,46 @@
+---
+published: true
+layout: default-theme-wet-boew-en
+title: Design decision 12 - Double H1 on a single page
+description: Confirming the possibility to use two H1s on a single page and conform to accessibility requirements
+modified: 2020-06-30
+---
+
+## Summary
+
+* **Author:** @GormFrank
+* **Expert:** @duboisp, @bsouster, @namjohn920, @GormFrank
+* **Status:** Draft
+* **Last updated:** 2020-06-30
+* **History:**
+	* (2020-06-30) Initial draft
+
+## Rational
+
+It was confirmed through research, discussions and accessiblity assessments that having two H1 tags on a single page is acceptable. By acceptable, it is meant that rightfully using double H1s on a page:
+* will not make you fail WCAG2.1 AA, as per an accessibility assessment led by the Accessibility, Accommodations & Adaptative Computer Technology (AAACT) Program team on February 26, 2020;
+* has the potential to improve your Search Engine Optimization (SEO) performances, as per a research led by the Principal Publisher team updated on February 18, 2020.
```

Comment:
It was presented by CRA team in January 9, 2020 and Principal Publisher has formally documented the evidence, February 18, 2020, to support the hypothesis.